src/charm.py

```diff
@@ -571,6 +593,32 @@ def _gnb_subnet_config_is_valid(self) -> bool:
     def _get_gnb_subnet_config(self) -> Optional[str]:
         return self.model.config.get("gnb-subnet")
 
+    def _is_cpu_compatible(self) -> bool:
+        """Returns whether the CPU meets requirements to run this charm.
+
+        Returns:
+            bool: Whether the CPU meets requirements to run this charm
+        """
+        return all(
+            required_extention in self._get_cpu_extensions()
```

Comment:
is there a typo there ? required_extension

Reply:
Good eye :)
Fixed.